website/docs/cloud-docs/api-docs/private-registry/tests.mdx

```diff
@@ -50,7 +51,7 @@ Alternatively, you can create a test with a pre-existing configuration version,
 
 The `tests` API endpoint has the following attributes.
 
-### Test run states
+### Test Run states
```

Comment:
nit: Should `states` also be capitalized because it is a noun here?

Reply:
yes, ty!